test/e2e/playwright.config.ts

```diff
@@ -12,16 +12,18 @@ require('dotenv').config();
  */
 export default defineConfig({
   expect: {
-    timeout: 60000,
+    timeout: 10000,
   },
-  timeout: 60000, //Set Test timeout to 1 minute
+  timeout: 30000, //Set Test timeout to 1 minute
```

Comment:
Should we remove this comment since it's no longer accurate?

Reply:
Yes, we should.